site/app/libraries/database/PostgresqlDatabaseQueries.php

```diff
@@ -1390,7 +1428,6 @@ public function getGradedGradeables(array $gradeables, $users = null, $teams = n
             // Always construct an instance even if there is no data
             $auto_graded_gradeable = new AutoGradedGradeable($this->core, $graded_gradeable, $row);
             $graded_gradeable->setAutoGradedGradeable($auto_graded_gradeable);
-
```

Comment:
remove this line deletion

Reply:
Still do remove this line deletion and the other one a hundred or so lines down.   Removing these blank lines reduces readability.